roles/nvidia_gpu_exporter/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Preflight
+  ansible.builtin.include_tasks:
+    file: preflight.yml
+  tags:
+    - nvidia_gpu_exporter_install
+    - nvidia_gpu_exporter_configure
+    - nvidia_gpu_exporter_run
+
+- name: Install
+  ansible.builtin.include_role:
+    name: prometheus.prometheus._common
+    tasks_from: install.yml
+  vars:
+    _common_local_cache_path: "{{ nvidia_gpu_exporter_local_cache_path }}"
+    _common_binaries: "{{ _nvidia_gpu_exporter_binaries }}"
+    _common_binary_install_dir: "{{ nvidia_gpu_exporter_binary_install_dir }}"
+    _common_binary_url: "{{ nvidia_gpu_exporter_binary_url }}"
+    _common_checksums_url: "{{ nvidia_gpu_exporter_checksums_url }}"
+    _common_system_group: "{{ nvidia_gpu_exporter_system_group }}"
+    _common_system_user: "{{ nvidia_gpu_exporter_system_user }}"
+    _common_binary_unarchive_opts: ['--strip-components=1']
+  tags:
+    - nvidia_gpu_exporter_install
```

Comment:
```suggestion
  tags:
    - nvidia_gpu_exporter
    - install
    - nvidia_gpu_exporter_install
```

Reply:
Same as last reply.
The tags are referred to node_exporter's.